scripts/release.js

```diff
@@ -1,8 +0,0 @@
-require('dotenv').config();
-const { ReleaseManager, ReleaseNoteManager } = require('@forestadmin/devops');
-
-const { DEVOPS_SLACK_TOKEN, DEVOPS_SLACK_CHANNEL } = process.env;
-const OPTIONS = { releaseIcon: '🌱', withVersion: true };
-
-new ReleaseManager(OPTIONS).create()
-  .then(() => new ReleaseNoteManager(DEVOPS_SLACK_TOKEN, DEVOPS_SLACK_CHANNEL, OPTIONS).create());
```

Comment:
How will the release note shared on Slack without this code?

Reply:
You are right. I decided (maybe I'm wrong) it would be done manually for the first releases: the current script has to be adapted to work. There is a dedicated package for posting to slack: https://github.com/juliuscc/semantic-release-slack-bot

I can work on this feature next week!